import/popups/PopupOpener.qml

```diff
@@ -62,6 +62,11 @@ Timer {
             return false
         }
 
+        if (data.text.length > 300) {
```

Comment:
I'm ok with this one. Let's just increase the length a bit. Maybe something like 1000? Any opinions?

Reply:
+ if (data.text &&...

Reply:
done